front/lib/models/assistant/agent.ts

```diff
@@ -177,6 +118,7 @@ AgentConfiguration.init(
     maxToolsUsePerRun: {
       type: DataTypes.INTEGER,
       allowNull: false,
+      // do we want to decrease this one to 0?
```

Comment:
I think we kind of want to remove this

Reply:
Should I keep the default value to 1 and just remove the comment, or remove the default value?

Reply:
Per Slack, removing the default value.